styles/_editorjs.scss

```diff
@@ -219,4 +219,18 @@
       background-color: transparent;
     }
   }
+  .codex-editor__redactor{
```

Comment:
<img width="292" alt="Screenshot 2023-02-21 at 16 03 43" src="https://user-images.githubusercontent.com/55176773/220327882-9ed55c3d-3853-4ad7-b278-5dbe164c731c.png">
scrollbar is not styled, you can get style of main element scrollbar